src/client/eppo-client.spec.ts

```diff
@@ -542,3 +546,133 @@ describe('EppoClient E2E test', () => {
     });
   });
 });
+
+describe(' EppoClient getAssignment From Obfuscated RAC', () => {
+  const storage = new TestConfigurationStore();
+  const globalClient = new EppoClient(storage);
+
+  beforeAll(async () => {
+    mock.setup();
+    mock.get(/randomized_assignment\/v3\/config*/, (_req, res) => {
+      const rac = readMockObfuscatedRacResponse();
+      return res.status(200).body(JSON.stringify(rac));
+    });
+    await init(storage);
+  });
+
+  afterAll(() => {
+    mock.teardown();
+  });
+
+  it.each(readAssignmentTestData())(
+    'test variation assignment splits',
+    async ({
+      experiment,
+      valueType = ValueTestType.StringType,
+      subjects,
+      subjectsWithAttributes,
+      expectedAssignments,
+    }: IAssignmentTestCase) => {
+      `---- Test Case for ${experiment} Experiment ----`;
+
+      if (experiment === 'targeting_rules_experiment') {
```

Comment:
are you able to test all experiment files?

Reply:
Oh yeah, this was the most pertinent test to getting the rule evaluation right, but all the test cases pass. I'll take out this if statement